akvo/akvo-lumen
# (DEPRECATED) (Changing approach ...) Transform caddisfly results to dataset columns

# (DEPRECATED) WIP
**change approach!!, instead of transforming caddisfly results in _"import"_  phase I'm trying to create a new tx type for that, follows up  => https://github.com/akvo/akvo-lumen/tree/issue-1571-txs **
This PR adds work on: [Better support Caddisfly data #1571]
- [ ] **Update release notes if necessary**
- [ ] do we need to load caddisfly schema tests from aws?? right now being loaded from packaged `resources`
- [ ] do we need to parse caddisfly test values to something different than `strings`?

- [ ] WIP Introduces a new id pattern for _child_ caddisfly new columns `(format "c%s%s" parent-id child-id)`

- [ ] We should set up a new updater version to keep compatibility with older imported datasets
- [ ] Is ok to **only** add this functionality to **flow_v3**? (does flow_v2 need to be extended too?)

# Steps for UI Testing 
- change `backend/dev/resources/dev.edn` :keycloak-url to point `test` environment (ask @tangrammer if you don't know how :) )
- click "+ Dataset" > select "Akvo Flow" > insert "uat1" > select survey "Caddisfly Test" > select sub-survey "Caddisfly Tests" > select "New form"
- that's all! (you can also verify the `update Dataset` functionality)
